provisioning_client/tools/dice_device_enrollment/dice_device_enrollment.c

```diff
@@ -17,7 +17,14 @@ static const char* const END_CERT_MARKER = "-----END CERTIFICATE-----";
 
 static char* get_user_input(const char* text_value, int max_len)
 {
-    char* result = malloc(max_len + 1);
+    int result_len = max_len + 1;
+    if (result_len <= 1)
+    {
+        (void)printf("invalid max_len size\r\n");
+        return NULL;
```

Comment:
It breaks our style of single point of return.

Reply:
line 31 already does an early return